### PR TITLE
Ensures that all dependencies are installed

### PR DESCRIPTION
**Pull request by igorkotof**
Testing:
[asciicast](https://asciinema.org/a/GSeUgYFQFrnCAjvu2Vj3gfoHP)

Testing #2 (06-12-2021):
image

### CODE REVIEW

**Comment by igorkotof** [comment]
[ICMK: Create `validate` task](https://trello.com/c/0z65hDsN/281-icmk-create-validate-task)

File: init.mk [hunk 1]
```diff
@@ -47,18 +47,42 @@ confirm:
 ########################################################################################################################
 # Core Dependencies
 GIT  ?= $(shell which git)
-DOCKER  ?= $(shell which docker)
+DOCKER ?= $(shell which docker)
+AWS-CLI ?= $(shell which aws)
```
Comment (AutomationD) [inline init.mk:51]: I would use an underscrore for `AWS_CLI` to keep it consistent with unix env-var naming

**Review by AutomationD — CHANGES_REQUESTED** [review]

**Comment by AutomationD** [comment]
@igorkotof any update on this? How hard would it be to fix it up the PR and wrap this up?

**Review by AutomationD — APPROVED** [review]

File: init.mk [hunk 2]
```diff
@@ -47,18 +47,42 @@ confirm:
...
+ssh-pub-key:
+ifeq (,$(wildcard ~/.ssh/id_rsa.pub))
+	@echo "\033[31m! SSH Public Key is not found here: ~/.ssh/id_rsa.pub . Please make sure that you have a key and pass it via '$(SSH_PUBLIC_KEY)' variable. \033[0m"
+else
+	@echo "\033[32m✔ SSH Public Key\033[0m"
 endif
 
-EXECUTABLES = $(GIT) $(DOCKER) aws
-K := $(foreach exec,$(EXECUTABLES),\
+#EXECUTABLES = $(GIT) $(DOCKER) aws
```
Comment (AutomationD) [inline init.mk:84]: Should we clean this up?

**Review by AutomationD — APPROVED** [review]
Looks good, thank you. just the comment cleanup.

**Review by kobrikx — APPROVED** [review]
lgtm

**Pull request merged**